Gemfile

```diff
@@ -1,12 +1,13 @@
-ruby "2.2.4"
+ruby RUBY_VERSION
```

Comment:
(I am sorry to confuse you with that docs link - that is a tricky way to "abuse" this feature.)